solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -451,6 +451,51 @@ contract LiquidityProtection is TokenHandler, Utils, Owned, ReentrancyGuard, Tim
         return addProtectedLiquidity(_owner, poolToken, _baseToken, poolTokenAmount / 2, _amount);
     }
 
+    /**
+     * @dev returns the single-side staking limits of a given pool
+     *
+     * @param _poolAnchor   anchor of the pool
+     * @return base maximum amount of base token liquidity that can be single-sided staked in the pool
+     * @return network maximum amount of base token liquidity that can be single-sided staked in the pool
+     */
+    function poolAvailableSpace(IConverterAnchor _poolAnchor) external view returns (uint256, uint256) {
+        // get the pool token
+        IDSToken poolToken = IDSToken(address(_poolAnchor));
+
+        // get the pool converter
+        ILiquidityPoolConverter converter = ILiquidityPoolConverter(payable(ownedBy(_poolAnchor)));
+
+        // get the base token
+        IERC20Token baseToken = converter.connectorTokens(0);
+        if (baseToken == networkToken) {
+            baseToken = converter.connectorTokens(1);
+        }
+
+        // get the reserve balances
+        (uint256 reserveBalanceBase, uint256 reserveBalanceNetwork) =
```

Comment:
Mayme `baseTokenReserveBalance` and `networkTokenReserveBalance`?

Reply:
We have the same naming issue cross system, sometimes using 'reserve', sometimes using `token` and sometimes using 'reserveToken'.
Due to ETH_RESERVE, it is probably wrong to use `token` in the general case.
Anyway, I don't think it's the right time and place to fix this issue, and fixing it here will just be a drop in the ocean. I actually copied that line from the function which imposes the limit (`addBaseTokenLiquidity`).